runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineContext.java

```diff
@@ -139,6 +141,9 @@ Converter createReader(
     Converter createWriter(
         ConverterConfig converter);
 
+    Validator supplyValidator(
+        ValidatorConfig converter);
+
```

Comment:
Please move this up next to other `supplyXXX` methods for guard, vault and catalog.
Also, shouldn't this be returning a `ValidatorHandler` (per core) instead of a `Validator` (global) given that it is on `EngineContext` which is per core?